openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/Table.spec.ts

```diff
@@ -92,4 +92,64 @@ test.describe('Table pagination sorting search scenarios ', () => {
 
     await expect(page.getByRole('tab', { name: 'Schema' })).toContainText('4');
   });
+
+  test.describe(
+    'should persist table pagination state while the user paginates and performs a browser back action',
+    () => {
+      test.beforeEach(async ({ page }) => {
+        await page.goto('/databaseSchema/sample_data.ecommerce_db.shopify');
+
+        await expect(page.getByTestId('databaseSchema-tables')).toBeVisible();
+      });
```

Comment:
why we are creating the separate describe, please try to include test in to main flow!